src/seedu/addressbook/AddressBook.java

```diff
@@ -486,9 +479,19 @@ private static String getMessageForPersonsDisplayedSummary(ArrayList<String[]> p
         final ArrayList<String[]> matchedPersons = new ArrayList<>();
         for (String[] person : getAllPersonsInAddressBook()) {
             final Set<String> wordsInName = new HashSet<>(splitByWhitespace(getNameFromPerson(person)));
-            if (!Collections.disjoint(wordsInName, keywords)) {
-                matchedPersons.add(person);
+            for(Iterator iterator = keywords.iterator(); iterator.hasNext();){
+                String keyword = iterator.next().toString().toLowerCase();
+                for(Iterator iterator1 = wordsInName.iterator();iterator1.hasNext();){
+                    String wordInName1 = iterator1.next().toString().toLowerCase();
+                    if(wordInName1.equals(keyword)){
+                        matchedPersons.add(person);
+                    }
+                }
             }
+
+            /*if (!Collections.disjoint(wordsInName, keywords)) {
```

Comment:
If the functionality is commented out means it is not in use anymore. Therefore, it shouldn't exist in the code base anymore.

You can use git history to look back for this instead.